my_project/tab_psy_chart/app_psy_chart.py

```diff
@@ -397,12 +403,12 @@ def update_psych_chart(
         fig.add_trace(
             go.Histogram2d(
                 x=df["DBT"],
-                y=df["hr"],
+                y=df_hr_multiply,
                 name="",
                 colorscale=var_color,
                 hovertemplate="",
                 autobinx=False,
-                xbins=dict(start=-50, end=60, size=1),
+                xbins=dict(start=-50, end=100, size=1),
```

Comment:
The xbins should be set larger. Considering when it changed to 'IP state'. The temperature will exceed the range.